dev/com.ibm.websphere.javaee.jaxws.2.3/bnd.bnd.disabled

```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
```

Comment:
Unfortunately, this bundle has to be disabled for now. The feature checker will fail if it is enabled, because since it's a spec API, the bundle cannot be declared in a private feature whose kind=noship.